apps/api/src/info-request/info-request.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '../prisma/prisma.service';
+import { CreateInfoRequestDto } from './dto/create-info-request.dto';
+import { UpdateInfoRequestDto } from './dto/update-info-request.dto';
+
+@Injectable()
+export class InfoRequestService {
+  constructor(public prisma: PrismaService){}
+
+  create(createInfoRequestDto: CreateInfoRequestDto) {
+    return this.prisma.infoRequest.create({data: createInfoRequestDto});
+  }
+
+  findAll() {
```

Comment:
@dplamenov  At the moment partially this service has been done in `SupportService`

https://github.com/podkrepi-bg/api/blob/b3ec755bb400e55715400080e3c6306d4c306a5f/apps/api/src/support/support.service.ts#L40-L56

It's kind of duplication as the creation of info request would be expected to do the same thing and at the moment it does two separate things.

Here are some ideas how to go further:

1. Remove the creation methods from `InfoRequestService`
2. Move the logic from `SupportService` to `InfoRequest` service and make them dependent
3. Something else that you might think of


Let's discuss the further steps to refactor this scenario so the code is located only on one place

Reply:
Can I delete the InfoRequestService and use SupportService at InfoRequestController?

Reply:
I committed changes but I am not sure is already ok.